packages/cli/src/databases/migrations/common/1736947513045-CreateTestCaseExecutionTable.ts

```diff
@@ -0,0 +1,47 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+const testCaseExecutionTableName = 'test_case_execution';
+
+export class CreateTestCaseExecutionTable1736947513045 implements ReversibleMigration {
+	async up({ schemaBuilder: { createTable, column } }: MigrationContext) {
+		await createTable(testCaseExecutionTableName)
+			.withColumns(
+				column('id').varchar(36).primary.notNull,
+				column('testRunId').varchar(36).notNull,
+				column('pastExecutionId').int, // Might be null if execution was deleted after the test run
+				column('executionId').int, // Execution of the workflow under test. Might be null if execution was deleted after the test run
+				column('evaluationExecutionId').int, // Execution of the evaluation workflow. Might be null if execution was deleted after the test run, or if the test run was cancelled
+				column('status').varchar().notNull,
```

Comment:
Could have a CHECK CONSTRAINT

Reply:
This is not settled yet, so we want to have some flexibility here and be able to change possible statuses quickly without db migration